stdlib/char.gr

```diff
@@ -153,11 +179,16 @@ let rec fromCode = (code: Number) => {
   result
 }
 
-// Returns the next valid Unicode character by code point. Fails if the input character is U+10FFFF.
-// @param char: Char - the input character
-// @returns Char
-let succ = (c) => {
-  let codePoint = code(c)
+/**
+ * Returns the next valid Unicode character by code point. Fails if the input character is U+10FFFF.
+ * 
+ * @param char: The input character
```

Comment:
```suggestion
 * @param char: The character
```

Reply:
@spotandjake I noticed above that you used `character` for some parameters. We should either choose `char` or `character` for each parameter. I'm leaning towards `char`.

Reply:
Hmms i think i might have used character on my first go a few months ago and just not changed it i will do that though.